src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -73,6 +75,22 @@ def validate_create_parameters(namespace):
         raise CLIError('--dns-prefix has no value')
 
 
+def validate_ip_ranges(namespace):
+    if not namespace.api_server_authorized_ip_ranges:
+        return
+
+    allow_all_traffic = "0.0.0.0/32"
+    for ip in namespace.api_server_authorized_ip_ranges.split(','):
+        if ip == allow_all_traffic:
+            continue
+        try:
+            ip = ip_network(ip)
+            if not ip.is_global:
```

Comment:
Is it easy to add a check for IPv6 address? We can error from CLI itself. If not, then we can rely on the check in RP sync.

Reply:
Added :)